internal/nexodus/policy_linux.go

```diff
@@ -201,6 +200,19 @@ func (nx *Nexodus) nfPermitProtoPortAddrV4(chain string, rule public.ModelsSecur
 					return err
 				}
 			}
+		} else if rule.FromPort != 0 && rule.ToPort != 0 {
```

Comment:
Here you use `!= 0`, but later in the patch you use `> 0`.

Ensuring it's a positive integer is probably better, but I won't hold the patch over it.